start_esp/test/testdata/expected_enable_strict_transport_security_nginx.conf

```diff
@@ -86,6 +83,9 @@ http {
 
 
 
+      # Enable HSTS (HTTP Strict Transport Security), 31536000 is 1 year.
+      add_header Strict-Transport-Security "max-age=31536000; includeSubdomains;" always;
+
```

Comment:
This does not solve the issue, nginx only applies the `add_header` directives in the 'deepest' level of indentation, ignoring all in the previous ones. Read [this](https://github.com/cloudendpoints/esp/issues/473#issuecomment-436558755).

Reply:
@guimarca The PR has been updated to place the add_header directives to other places. Can you check again?